src/components/pages/Profile/EditProfile.js

```diff
@@ -2,51 +2,39 @@ import React, { useState } from 'react';
 import { Form, Input, Button, Radio, Modal, TreeSelect } from 'antd';
 import '../../../styles/styles.css';
 import { connect } from 'react-redux';
+import useForms from '../../../hooks/useForms';
+
+const initialValues = {
+  first_name: 'Hal',
+  last_name: 'Jordan',
+  email: 'greenguy123@gmail.com',
+  location: 'Earth',
+  company: 'Bloom Tech, SWE',
+  tech_stack: 'React',
+  commitment: 'Pair Programming',
+};
 
 function EditProfile({ userInfo }) {
-  // Grab initial values from profile component
-  const initialValues = {
-    first_name: 'Hal',
-    last_name: 'Jordan',
-    email: 'greenguy123@gmail.com',
-    location: 'Earth',
-    company: 'Bloom Tech, SWE',
-    tech_stack: 'React',
-    commitment: 'Pair Programming',
-  };
+  const [formValues, handleChange, clearForm] = useForms(initialValues);
 
   const [ModalOpen, setModalOpen] = useState(false);
-  // eslint-disable-next-line no-unused-vars
-  const [radio, setRadio] = useState(initialValues);
 
-  //// Event Handlers
+  const showModal = () => setModalOpen(true);
 
-  const showModal = () => {
-    setModalOpen(true);
-  };
-
-  const handleCancel = () => {
-    setModalOpen(false);
-  };
+  const handleCancel = () => setModalOpen(false);
```

Comment:
I definitely like this change. Anytime we can reduce three lines into one is a plus!